modules/local/prepare_shiny.nf

```diff
@@ -29,7 +29,8 @@ process PREPARE_SHINY {
     cp $table shiny_data.csv
     cp -r $app/* .
     rm $app
-    echo "docker run -v .:/app --network=host $docker_url $bash_command" > run.sh
+    echo -n 'docker run -v \$PWD:/app --network=host ' > run.sh
```

Comment:
Have you verified that this works in 2 lines?

Reply:
Yes :) I had to do it because i need the single ' on the first line and the " on the second